### PR TITLE
Create assembly page

### PR DESCRIPTION
**Pull request by okdistribute**

### CODE REVIEW

File: site/meetup.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+body_class: workshop
+title: DOTS Assembly
+---
+
+# Join our monthly community meetup!
+
+We invite comments, questions, and new ideas regarding design patterns 
+in decentralization and related technologies.
+
+Every second Monday of the month at 16:00 UTC / 18:00 Berlin / 9:00 California.
```
Comment (cafca) [inline site/meetup.md:13]: Do US-Americans understand "9:00" without the "am"?

**Comment by cafca** [comment]
I added links to the assembly page from landing and contribute? I also added headers to the itemized contribution thingies. Let me know what you think.

**Review by cafca — APPROVED** [review]

**Comment by cafca** [comment]
@okdistribute for some reason I can't request a review from you 🤷‍♂️

**Comment by cafca** [comment]
Where's tender hermann? 🤔

**Comment by okdistribute** [comment]
Looks great!

**Pull request merged**